decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -4,7 +4,41 @@
     <div class="static__message">
       <h1 class="heading1 page-title"><%= t(".title") %></h1>
       <p><%= t(".try_later") %></p>
+      <div rows="6" id="input-text">
+        <div class="card--list__item">
+          <div class="card--list__text">
+            <strong class="margin-right-2"><%= t(".user_id") %></strong>
+            <%= user = try(:current_user)&.id || t(".unknown") %>
+          </div>
+        </div>
+        <div class="card--list__item">
+          <div class="card--list__text">
+            <strong class="margin-right-2"><%= t(".date_and_time") %></strong>
+            <%= date_and_time = l(Time.current, format: "%d/%m/%Y - %H:%M") %>
```

Comment:
Thanks for the comments. Just to sum up, I think the current solution would help tracking down the reported issue for now with combining all of the information provided.

Reply:
The problem also exists for the current format of date and time.

In case the exception is thrown at the very last second of the minute, the date+time shown on this section still does not match the minute when the exception was thrown.

I think most of the times this wouldn't be an issue if we correct the way we show the "reference" (see the other comment about that) but this can be still misleading. But the problem is (as stated) that I don't think there's an easy way to get the exact same date+time value as is visible in the logs due to the reasons explained above. I can look further into that if I can figure out some way.